sites/public/src/pages/applications/start/choose-language.tsx

```diff
@@ -100,7 +100,7 @@ const ApplicationChooseLanguage = () => {
         listingName={listing?.name}
         heading={t("application.chooseLanguage.letsGetStarted")}
         progressNavProps={{
-          currentPageSection: 0,
+          currentPageSection: 1,
```

Comment:
<img width="413" alt="Screenshot 2024-02-06 at 1 09 23 PM" src="https://github.com/bloom-housing/bloom/assets/65367387/0e0fb20a-7eee-43f4-b5df-d74db94570c9">
A separate bug fix sneaking in here for this issue